app/code/Magento/NewRelicReporting/Plugin/HttpPlugin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\NewRelicReporting\Plugin;
+
+use Magento\Framework\App\Http;
+use Magento\NewRelicReporting\Model\Config;
+use Magento\NewRelicReporting\Model\NewRelicWrapper;
+
+class HttpPlugin
+{
+    /**
+     * @var Config
+     */
+    protected $config;
+
+    /**
+     * @var NewRelicWrapper
+     */
+    protected $newRelicWrapper;
+
+    /**
+     * @param Config $config
+     * @param NewRelicWrapper $newRelicWrapper
+     */
+    public function __construct(
+        Config $config,
+        NewRelicWrapper $newRelicWrapper
+    ) {
+        $this->config = $config;
+        $this->newRelicWrapper = $newRelicWrapper;
+    }
+
+    public function beforeCatchException(Http $subject, $bootstrap, $exception)
```

Comment:
Could you add types for $bootstrap and $exception? Also physic block is missing

Reply:
@ihor-sviziev this is done.